_analyzers/supported-analyzers/standard.md

```diff
@@ -26,41 +29,44 @@ PUT /my_standard_index
     "properties": {
       "my_field": {
         "type": "text",
-        "analyzer": "standard"  
+        "analyzer": "standard"
       }
     }
   }
 }
 ```
 {% include copy-curl.html %}
 
+---
+
 ## Parameters
 
-You can configure a `standard` analyzer with the following parameters.
+The `standard` analyzer supports the following optional parameters:
 
-Parameter | Required/Optional | Data type | Description
-:--- | :--- | :--- | :--- 
-`max_token_length` | Optional | Integer | Sets the maximum length of the produced token. If this length is exceeded, the token is split into multiple tokens at the length configured in `max_token_length`. Default is `255`.
-`stopwords` | Optional | String or list of strings | A string specifying a predefined list of stopwords (such as `_english_`) or an array specifying a custom list of stopwords. Default is `_none_`.
-`stopwords_path` | Optional | String | The path (absolute or relative to the config directory) to the file containing a list of stop words.
+| Parameter | Data type | Default | Description |
+|:----------|:-----|:--------|:------------|
+| `max_token_length` | Integer | `255` | Sets the maximum length of a token before it is split. |
+| `stopwords` | String or list of strings | None | A list of stopwords or a predefined stopword set like `_english_` to remove during analysis. |
```

Comment:
Is there a list of the predefined stopwords anywhere that we can link to?

Reply:
I'm not aware of such list, as they are built into lucene directly, see example [danish](https://github.com/apache/lucene/blob/main/lucene/analysis/common/src/resources/org/apache/lucene/analysis/snowball/danish_stop.txt)